04.class/memo_app.js

```diff
@@ -96,8 +96,7 @@ export class MemosApp {
     })();
   }
 
-  extractFirstLine(baseData) {
-    const lines = baseData.split("\n");
-    return lines[0];
+  extractFirstLine(lines) {
```

Comment:
linesは配列ではなく、文字列ですよね？
36行目で`this.baseData[uuid] = memo;`としていることを考えると、

```suggestion
  extractFirstLine(memo) {
```

とした方が良いと思います。
そして、もしlinesを登場させるなら、`const lines = memo.split("\n");`になりそうですね。